Resources/Prototypes/Entities/Clothing/Shoes/misc.yml

```diff
@@ -148,3 +148,23 @@
     price: 75
   - type: Tag
     tags: [ ]
+   
+- type: entity
+  parent: ClothingShoesBase
+  id: ClothingShoesJumpBoots
+  name: jump boots
+  description: Technological boots used by explorers of new worlds from various unwanted obstacles.
+  components:
+  - type: Sprite
+    sprite: Clothing/Shoes/Boots/jumpboots.rsi
+    layers:
+    - state: icon
+  - type: Clothing
+    sprite: Clothing/Shoes/Boots/jumpboots.rsi
+  - type: Jump
+    jumpPower: 3
+    soundJump: /Audio/Effects/stealthoff.ogg
```

Comment:
> why add both sound files if only one is used?

I wrote in Technical details that it is possible not only to add to clothes but also to characters, so that someone in the future could use this system for any developments on the type of abilities of antags or etc. That's why there are two sounds as well, one that applies only to these boots, and one that is separate just as the foundation of the system.